Modules/IO/NIFTI/test/itkNiftiImageIOTest3.cxx

```diff
@@ -295,5 +300,10 @@ itkNiftiImageIOTest3(int argc, char * argv[])
   success |= TestImageOfVectors<float, 4, 4>(std::string("testVectorImage_float_4_4.nii.gz"));
   success |= TestImageOfVectors<double, 3, 3>(std::string("testVectorImage_double_3_3.nii.gz"));
 
+  // Test reading/writing as displacement field (NIFTI intent code = 1006)
+  success |=
+    TestImageOfVectors<double, 3, 1>(std::string("testDispacementImage_double_3_1.nii.gz"), std::string("1006"));
+  success |= TestImageOfVectors<float, 3, 1>(std::string("testDisplacementImage_float.nii.gz"), std::string("1006"));
```

Comment:
"testDisplacementImage_float.nii.gz", should that be "testDisplacementImage_float_3_1.nii.gz"? No problem to me, just wondering, because the other one has "_3_1" in the file name.

Reply:
Good catch. It should not affect functionality, but I'll rename for consistency.